packages/google_maps_flutter/google_maps_flutter/example/pubspec.yaml

```diff
@@ -18,6 +18,8 @@ dependencies:
     # The example app is bundled with the plugin so we use a path dependency on
     # the parent directory to use the current plugin's version.
     path: ../
+  google_maps_flutter_platform_interface:
+    path: ../../google_maps_flutter_platform_interface
```

Comment:
We can't merge with this code, so this PR will need some revisiting after #2653 lands!